mrp_multi_level/readme/CONFIGURE.md

```diff
@@ -0,0 +1,9 @@
+### MRP Areas
+
+- Go to *Manufacturing \> Configuration \> MRP Areas* and define or edit
+  any existing area. You can specify the working hours for every area.
+
+### Product MRP Area Parameters
+
+- Go to *Manufacturing \> Master Data \> Product MRP Area Parameters*
```

Comment:
Little change here:
```suggestion
- Go to *Manufacturing \> Products \> Product MRP Area Parameters*
```

Reply:
@JasminSForgeFlow this one is pending to adjust